src/Converter.cpp

```diff
@@ -0,0 +1,93 @@
+#include "Converter.hpp"
+#include "frame_helper/FrameHelper.h"
+#include "base-logging/Logging.hpp"
+#include "opencv2/core/core.hpp"
+#include "opencv2/highgui/highgui.hpp"
+
+namespace log2video
+{
+
+Converter::Converter()
+{
+
+}
+
+void Converter::convert(std::string input_file, std::string stream, std::string output_file)
```

Comment:
In C++, objects used as input arguments are usually passed as constant references (it is faster this way)

Reply:
:+1: The rule of thumb is to use const ref for everything but fundamental types.